src/app/design/guidelines/user-assistance/index.html

```diff
@@ -71,5 +113,49 @@
 			</div>
 		</stache-column>
 	</stache-row>
-  
+
+	<stache-page-anchor>
+		Popovers
+	</stache-page-anchor>
+
+	<stache-row>
+		<stache-column screenSmall="6">
+			<div class="sky-subsection-heading">Usage</div>
+			<p>
+				<a stacheRouterLink="/components/popover">Popovers</a> are a great way to guide users toward a subtle but important element on the page or modal. Users can display popovers when they interact with their trigger elements, and you can also display popovers programmatically when a page loads. It is appropriate to use popover help in cases where:
+			</p>
+			<ul>
+				<li>
+					An element is important but easily overlooked. For example, use a popover for a new filter option, a view switcher, or an isolated context menu.
+				</li>
+				<li>
+					An element requires special attention when it loads because user research indicates a problem with discoverability.
+				</li>
+			</ul>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/userassistance/popover-example.png" />
+			</div>
+		</stache-column>
+	</stache-row>
+
+	<stache-page-anchor>
+		Preloaded Help Widget
+	</stache-page-anchor>
+
+	<stache-row>
+		<stache-column screenSmall="6">
+			<div class="sky-subsection-heading">Usage</div>
+			<p>
+				You can preload the Help Widget to display user assistance that is tailored to the first time that users load a page. Use this technique to introduce users to new or novel functionality. Do not use it to walk users through processes.
```

Comment:
What do you think about adding a note for Blackbaud developers that links to them to https://docs.blackbaud.com/bb-help-docs/ ? I know there's also a link to those docs from the Help inline component, but because so many people have mentioned they didn't even know those docs existed, I'm inclined to include it as much as possible, when applicable of course.

Reply:
I think that's a great idea. Good call.